api/Element.json

```diff
@@ -3786,6 +3786,210 @@
           }
         }
       },
+      "scroll": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/scroll",
+          "support": {
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": false
+            },
+            "edge_mobile": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
+            "samsunginternet_android": {
+              "version_added": true
+            },
+            "webview_android": {
+              "version_added": true
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "ScrollToOptions": {
+          "__compat": {
+            "description": "Can accept <code>ScrollToOptions</code> dictionary as parameter.",
```

Comment:
We try to keep the descriptions really short and concise. Unlike `notes` they shouldn't be full sentences. Descriptions here are actually like a "label" than a "description". (changing this in the schema is possible but probably breaks with data consumers now). Also in the table on MDN there is not much space. So, I would suggest to make this `"<code>ScrollToOptions</code> parameter"` here and throughout this PR.

Reply:
Makes sense; changes made. Thanks @Elchi3 !